Automattic/jetpack
# Hide some admin menu items for P2s

See D67475-code

#### Changes proposed in this Pull Request:

* This hides P2 Pattern menu items not necessary in Calypso.

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:






* Verify that there is no "P2 Patterns" menu item on workspaces.
* Verify that there is a "P2 Patterns" menu item on P2s.
* Verify that there is no "tags" submenu item under "P2 Patterns" on P2s.
